src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenLocalFunctionTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
```

Comment:
@dpoeschl yeah it's the BOM. we should fix that as @sharwell just did a change to add the BOM, this appears to be undoing that. 

Guess codeflow hides this difference which is why I missed it.